cmd/brew-test-bot.rb

```diff
@@ -369,10 +369,6 @@ def diff_formulae(start_revision, end_revision, path, filter)
         pr = $1
         @url = "#{git_url}/pull/#{pr}"
         @hash = nil
-      # Use Travis CI pull-request variables for pull request jobs.
-      elsif travis_pr
```

Comment:
Can you remove the `travis_pr` variable that's set above and the `Travis` from `# Handle a URL being passed on the command-line or through Jenkins/Travis environment variables` below? Thanks ❤️